snuba/cli/devserver.py

```diff
@@ -143,14 +143,11 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                 "metrics-consumer",
                 [
                     "snuba",
-                    "multistorage-consumer",
-                    "--storage=metrics_counters_buckets",
-                    "--storage=metrics_distributions_buckets",
-                    "--storage=metrics_buckets",
+                    "consumer",
                     "--storage=metrics_raw",
                     "--auto-offset-reset=latest",
                     "--log-level=debug",
-                    "--consumer-group=metrics_group",
+                    "--consumer-group=snuba-metrics-consumers",
```

Comment:
also changing this to be the same as what we have for prod https://github.com/getsentry/ops/blob/master/k8s/services/snuba/_deployment.metrics-consumer.yaml#L9